Pipelines/Templates/tests.yaml

```diff
@@ -14,61 +9,33 @@ parameters:
     - StandaloneWindows64
     - WSAPlayer
     - Android
-
+  default: StandaloneWindows64
+    
 - name: PathToProject
   type: string
+  default: ''
 
 - name: AdditionalArguments
   type: string
   default: ''
 
 steps:
-# No edit mode tests (yet!)
-# - pwsh: |
-    # Write-Host "======================= EditMode Tests ======================="
```

Comment:
Could you restore the commented out edit mode test runner please? Probably with the same `TempDirectory` -> `ArtifactStagingDirectory` change you made for play mode tests